packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -0,0 +1,160 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
+
+import 'system_channels.dart';
+
+/// The [PlatformViewRegistry] responsible for generating unique identifiers for platform views.
+final PlatformViewRegistry platformViewRegistry = PlatformViewRegistry._instance();
+
+/// A registry responsible for generating unique identifier for platform views.
+///
+/// A Flutter application has a single [PlatformViewRegistry] which can be accesses
+/// through the [platformViewRegistry] getter.
+///
+/// See also:
+///   * [PlatformView], a widget that shows a platform view.
+class PlatformViewRegistry {
+  PlatformViewRegistry._instance() {
+    _nextPlatformViewId = 0;
```

Comment:
Nit: initialize on the field `int _nextPlatformViewId = 0`

Reply:
Done, is there anything about this in the style guide?